subsys/bluetooth/controller/ll_sw/ull_llcp.c

```diff
@@ -249,11 +252,11 @@ void llcp_tx_flush(struct ll_conn *conn)
  * LLCP Procedure Creation
  */
 
-static struct proc_ctx *create_procedure(enum llcp_proc proc)
+static struct proc_ctx *create_procedure(enum llcp_proc proc, struct mem_pool *owner)
```

Comment:
Referring to my comment from above, I would not name the `mem_pool` as `owner`. In this context it is a memory pool. 
Look at line below `ctx = proc_ctx_acquire(owner);`. Would it be easier to understand if that is `ctx = proc_ctx_acquire(ctx_pool)`?